e2e_tests/application_test.py

```diff
@@ -43,43 +39,51 @@ def test_create_business_application(self):
-        self.assertTrue(response.data.type == "businessApplication")
+        return self.client.applications.create(request)
+
+    def test_create_individual_application(self):
+        app = self.create_individual_application()
+        self.assertTrue(app.data.type == "individualApplication")
+
+    def test_create_business_application(self):
+        app = self.create_business_application()
+        self.assertTrue(app.data.type == "businessApplication")
 
     def test_get_applications(self):
-        token = os.environ.get("token")
-        client = Unit("https://api.s.unit.sh", token)
-        response = client.applications.get("72996")
+        response = self.client.applications.get("72996")
         self.assertTrue(response.data.type == "businessApplication" or response.data.type == "individualApplication")
 
     def test_list_applications(self):
-        token = os.environ.get("token")
-        client = Unit("https://api.s.unit.sh", token)
-        response = client.applications.list()
+        response = self.client.applications.list()
         for app in response.data:
             self.assertTrue(app.type == "businessApplication" or app.type == "individualApplication")
 
     def test_upload_application_document(self):
-        token = os.environ.get("token")
-        client = Unit("https://api.s.unit.sh", token)
         app = self.create_individual_application()
         doc_id = app.included[0].id
         with open("../sample.pdf", 'rb') as file:
             request = UploadDocumentRequest(app.data.id, doc_id, file.read(), "pdf")
-        response = client.applications.upload(request)
+        response = self.client.applications.upload(request)
         self.assertTrue(response.data.type == "document")
 
     def test_upload_application_back_document(self):
-        token = os.environ.get("token")
-        client = Unit("https://api.s.unit.sh", token)
         app = self.create_individual_application()
         doc_id = app.included[0].id
         with open("../sample.pdf", 'rb') as file:
             request = UploadDocumentRequest(app.data.id, doc_id, file.read(), "pdf", True)
-        response = client.applications.upload(request)
+        response = self.client.applications.upload(request)
         self.assertTrue(response.data.type == "document")
 
+    def test_update_individual_application(self):
+        app = self.create_individual_application()
+        updated = self.client.applications.update(PatchApplicationRequest(app.data.id, tags={"patch": "test-patch"}))
+        self.assertTrue(updated.data.type == "individualApplication")
+
+    def test_update_bussiness_application(self):
+        app = self.create_business_application()
+        updated = self.client.applications.update(PatchApplicationRequest(app.data.id, "businessApplication",
+                                                                          tags={"patch": "test-patch"}))
+        self.assertTrue(updated.data.type == "businessApplication")
```

Comment:
@axshani Should we test that we really added the tags or is that something we havent done in other tests as well?